Tools/autotest/sim_vehicle.py

```diff
@@ -842,6 +855,14 @@ def generate_frame_help():
                      action='store_true',
                      default=False,
                      help="use gdb for debugging ardupilot (no auto-start)")
+group_sim.add_option("-X", "--lldb",
```

Comment:
Could we do without the "-X" and "-x" here, please?

Reply:
Yeah, "b" was taken. Any suggestions? Or just have the long options?

Reply:
Ok removed